tsconfig-dist.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "es2015",
```

Comment:
`es2015` is not *absolutely* necessary to make AOT work, but **it is** needed if you want to move on with tree-shaking. See e.g. [angular docs](https://angular.io/docs/ts/latest/cookbook/aot-compiler.html). Anyway, AOT by itself should work also with `commonjs`, so if you feel like you might want to change that. Until now, we're not having any issue with webpack 2.x.